model_one/__init__.py

```diff
@@ -1,24 +1,18 @@
 from model_one.cli import (
-    create_classifier,
-    create_generator,
-    get_model,
-    models,
-    train_generator,
-    train_classifier,
+    BeyondmlModelException,
+    BeyondmlModelStatus,
+    BeyondmlModelType,
     BeyondmlModel,
-    api_url,
+    BeyondmlAPI,
     API_KEY
 )
 
 __all__ = [
-    'model_one',
-    'API_KEY',
-    'api_url',
-    'create_classifier',
-    'create_generator',
-    'get_model',
-    'models',
-    'train_generator',
-    'train_classifier',
-    'BeyondmlModel'
+    "model_one",
```

Comment:
I believe that sticking with using `"` instead of `'` wherever it is possible is the right way